src/song.cpp

```diff
@@ -160,6 +167,13 @@ void		Song::init()
 		year = "";
 	}
 
+	/* original year from original date */
```

Comment:
I see your point. But for your purposes would sorting by originaldate and yet displaying only date (reissue) do the trick? You could display the originalyear as well as the year in your topbar, as I'm doing in my config. (I'd rather only show the originalyear if originaldate is different from date, but that'd require conditionals which we don't have.)